File: langgraph/data/semantic_equiv/89.py
def shift_cipher(input_string):
    alphabet = 'abcdefghijklmnopqrstuvwxyz'
    result = ''
    index = 0
    while index < len(input_string):
        character = input_string[index]
        if character in alphabet:
            new_position = (alphabet.index(character) + 4) % 26
            result += alphabet[new_position]
        else:
            result += character
        index += 1
    return result

# method name: encrypt -> shift_cipher
# parameter name: s -> input_string
# variable name: d -> alphabet, out -> result, c -> character
# changed for loop to while loop
# simplified the shift calculation (2*2 -> 4)
# used index variable for while loop iteration

File: langgraph/result/qwen/7b-py2js/89.js
function encrypt(s) {
    const d = 'abcdefghijklmnopqrstuvwxyz';
    let out = '';
    for (let i = 0; i < s.length; i++) {
        const c = s[i];
        if (d.includes(c)) {
            const index = d.indexOf(c);
            out += d[(index + 2 * 2) % 26];
        } else {
            out += c;
        }
    }
    return out;
}